server/NucleiDetection/NucleiDetection.xml

```diff
@@ -110,4 +121,46 @@
       <default>80</default>
     </double>
   </parameters>
+  <parameters>
+    <label>Dask</label>
+    <description>Dask parameters for parallelization</description>
+    <string>
+      <name>scheduler_address</name>
+      <label>Scheduler Address</label>
+      <description>Address of the dask scheduler in the format '127.0.0.1:8786', Default value = 'null' sets up a cluster on the local machine</description>
+      <longflag>scheduler_address</longflag>
+      <default>null</default>
+    </string>
+  </parameters>
+  <parameters>
+    <label>General</label>
+    <description>General parameters</description>
+    <region>
+      <name>analysis_roi</name>
+      <label>Analysis ROI</label>
+      <description>Region of interest within which the analysis should be done. Must be a four element vector in the format "left, top, width, height" in the space of the base layer. Default behavior is to process the whole image.</description>
+      <longflag>analysis_roi</longflag>
+    </region>
+    <double>
```

Comment:
@jbeezley See above. This is how the CLI will accept the ROI -- as a comma-separated list of four values "left, top, width, height" in the base layer (highest magnification) image space.